include/tl2cgen/detail/compiler/ast/builder.h

```diff
@@ -33,7 +33,7 @@ class ASTBuilder {
   ASTBuilder() : main_node_(nullptr) {}
 
   /* \brief Initially build AST from model */
-  void BuildAST(treelite::Model const& model);
+  void BuildAST(treelite::Model const& model, bool thresh_as_int);
```

Comment:
Adding the new option `thresh_as_int` is fine for now. But if we were to add more options to BuildAST, we should consider passing around the `CompilerParam` struct instead, to avoid having too many function arguments.